test/suites/twitter.delete.tweet.js

```diff
@@ -0,0 +1,183 @@
+const Promise = require('bluebird');
+const assert = require('assert');
+const sinon = require('sinon');
+const { merge } = require('lodash');
+const { EventEmitter } = require('events');
+
+
+describe('twitter delete from stream', function testSuite() {
+  this.retries(20);
+
+  const Social = require('../../src');
+  const STATUSES_TABLE = 'statuses';
+
+  const payload = {
+    accounts: [
+      { account_id: '111' },
+      { account_id: '222' },
+      { account_id: '333' },
+    ],
+    register: {
+      internal: 'test@test.ru',
+      network: 'twitter',
+      accounts: [
+        { username: 'evgenypoyarkov' },
+        { id: '2533316504', username: 'v_aminev' },
+      ],
+    },
+    read: {
+      filter: {
+        account: 'v_aminev',
+      },
+    },
+    mockDeleteData: {
+      status: {
+        id: 558603122315382784,
+        id_str: '558603122315382784',
+        user_id: 214194508,
+        user_id_str: '214194508',
+      },
+    },
+    twitterStub: {
+      created_at: 'Thu Apr 06 15:24:15 +0000 2017',
+      id_str: '100',
+      text: '1) Today we\u2019re sharing our vision for the future of the Twitter API platform!\nhttps://t.co/XweGngmxlP',
+      user: {
+        id: 1,
+        name: 'Twitter Dev',
+        screen_name: 'TwitterDev',
+        location: 'Internet',
+        url: 'https://dev.twitter.com/',
+        description: `Your official source for Twitter Platform news, updates & events.
+        Need technical help? Visit https://twittercommunity.com/ \u2328\ufe0f #TapIntoTwitter`,
+      },
+      place: {
+      },
+      entities: {
+        hashtags: [
+        ],
+        urls: [
+          {
+            url: 'https://t.co/XweGngmxlP',
+            unwound: {
+              url: 'https://cards.twitter.com/cards/18ce53wgo4h/3xo1c',
+              title: 'Building the Future of the Twitter API Platform',
+            },
+          },
+        ],
+        user_mentions: [
+        ],
+      },
+    },
+  };
+
+
+  let service;
+  const ee = new EventEmitter();
+  ee.destroy = () => (ee.removeAllListeners());
+  let onDeleteSpy; // spy for 'delete' event handler
+  let onDataSpy; // spy for 'data' event handler
+  let delTwitStatusStub; // stub for markDelete() method from src/services/storage/twitter-statuses.js
+
+  let setFollowStub; // stub for this.setFollowing() in src/services/twitter.js
+  let fillAccIdsStub; // stub for this.fillAccIds() in src/services/twitter.js
+  let listenStub; // stub for this.listen in src/services/twitter.js
+
+  let allStubs; // all the stubs storage to restore them at the end
+  const tweetId = payload.twitterStub.id_str;
+
+  function initSpiesNStubs() {
```

Comment:
`initSpiesNStubs` - More human readable function name will be better

Reply:
initSpiesAndStubs()?